getting_started_moa/creating-first-moa-cluster.adoc

```diff
@@ -0,0 +1,46 @@
+:context: creating-first-moa-cluster
+[id="creating-first-moa-cluster"]
+= Getting started with {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+This guide walks through setting up your first {product-title} cluster using `moactl`.
+
+If you have already installed the required prerequisites, here are the commands you need to create a cluster.
+
+[source, bash]
+----
+## Configures your AWS account and ensures everything is setup correctly
+$ moactl init
+
+## Starts the cluster creation process (~30-40minutes)
+$ moactl create cluster --name <my-cluster-name>
+
+## Watch your logs as your cluster creates
+$ moactl logs cluster <my-cluster-name> --watch
+
+## Connect your IDP to your cluster
+$ moactl create idp --cluster <my-cluster-name>  --interactive
+
+## Promotes a user from your IDP to admin level
+$ moactl create user --cluster <my-cluster-name> --dedicated-admins <admin-username>        
+
+## Checks if your install is ready (look for State: Ready),
+## and provides your Console URL to login to the web console.
+$ moactl describe cluster <my-cluster-name>
+----
+
+If you are just starting out, or if you get stuck, consult the rest of this guide for more information on each step.
+
```

Comment:
I would combine these two sentences. 

"For additional help, consult the rest of this tutorial for guided steps on running an Amazon Red Hat OpenShift cluster on your AWS account. " ?